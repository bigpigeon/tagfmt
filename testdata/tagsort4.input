//tagfmt -s -sw "json=2|yaml=1|toml=1|desc=-1" -so "toml|yaml|json"
package main
type Example struct {
	Data string `desc:"some inuse data" yaml:"data" toml:"data" binding:"required" json:"data" `
}