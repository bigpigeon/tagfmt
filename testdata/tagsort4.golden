//tagfmt -s -sw "json=2|yaml=1|toml=1|desc=-1" -so "toml|yaml|json"
package main

type Example struct {
	Data string `json:"data" toml:"data" yaml:"data" binding:"required" desc:"some inuse data"`
}
